spec/authorization_spec.rb

```diff
@@ -273,6 +273,91 @@ def to_params(*args, **kwargs, &block)
     end
   end
 
+  if Gem::Version.new(ActionPack::VERSION::STRING) >= Gem::Version.new("8.0.0")
```

Comment:
Looks like JRUBY doesn't support this version? I don't know if this is a good approach though :)

Reply:
It's either this or checking for if the specific API method exists. JRuby 9.3 is EOL anyway: https://github.com/jruby/jruby/wiki/Roadmap